packages/opencensus-core/src/stats/model/bucket-boundaries.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {BucketBoundaries} from './types';
+
+export class StatsBucketBoundaries implements BucketBoundaries {
+  private bucketBoundaries: number[];
+
+  constructor(bucketBoundaries: number[]) {
+    this.bucketBoundaries = bucketBoundaries.sort();
```

Comment:
`bucketBoundaries.slice().sort()` -- better not mutate the input.

Reply:
File removed